sdk/src/test/java/dev/cdevents/custom/resource/Nested.java

```diff
@@ -0,0 +1,95 @@
+
+package dev.cdevents.custom.resource;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import javax.annotation.Generated;
+import java.util.ArrayList;
+import java.util.List;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonPropertyOrder({
```

Comment:
I know there's some reluctance to using Lombok, but the more I see this type of code... Im like dude, lombok will clean this up, even if it is a test class it'll make things much easier to read which is the **most** important aspect of code, imo.

Like we added 6 classes here. Each 100+ lines. With lobok it'd be maybe 20 lines.

@aalmiray